css/index.scss

```diff
@@ -351,8 +351,8 @@
   }
 
   button {
-    background-color: #3bbdc4;
-    border-color: #3bbdc4;
+    background-color: #193d54;
```

Comment:
can you comment a bit more on this button color change? do you have a screenshot of what it would look like? 

i'm hesitant to change the default color scheme, so it would be good to understand the visual impact of this change a bit more :)

Reply:
# before
<img width="585" alt="Screenshot 2022-08-17 at 11 19 38 AM" src="https://user-images.githubusercontent.com/59417546/185044162-5ffb3207-9333-490e-a16a-c4397ac464c4.png"> 

# after 

<img width="599" alt="Screenshot 2022-08-17 at 11 19 30 AM" src="https://user-images.githubusercontent.com/59417546/185044140-1577cb14-a16f-4a54-ba19-731d45a10a3e.png">

Reply:
made these changes to improve the contrast ratio for better visibility and improving SEO

Reply:
thanks for sharing this screenshot to help us visualize the button change more easily!

Reply:
should i keep this or revert it? this didn't make a huge improvement itself but small improvements contribute to bigger improvements